UI/window-basic-auto-config.cpp

```diff
@@ -566,7 +566,8 @@ void AutoConfigStreamPage::UpdateKeyLink()
 	if (serviceName == "Twitch") {
 		streamKeyLink =
 			"https://www.twitch.tv/broadcast/dashboard/streamkey";
-	} else if (serviceName == "YouTube / YouTube Gaming") {
+	} else if ((serviceName == "YouTube-RTMP") or
```

Comment:
This change needs to be pushed to add-youtube-hls-streaming branch, and keep the link to the old url.